src/main/java/it/gov/pagopa/pu/bff/controller/OrganizationController.java

```diff
@@ -1,28 +1,28 @@
 package it.gov.pagopa.pu.bff.controller;
 
-import it.gov.pagopa.pu.bff.controller.generated.BrokersApi;
-import it.gov.pagopa.pu.bff.dto.generated.ConfigFE;
+import it.gov.pagopa.pu.bff.controller.generated.OrganizationsApi;
+import it.gov.pagopa.pu.bff.dto.generated.OrganizationDTO;
 import it.gov.pagopa.pu.bff.security.SecurityUtils;
-import it.gov.pagopa.pu.bff.service.organization.BrokerServiceImpl;
-import lombok.extern.log4j.Log4j2;
+import it.gov.pagopa.pu.bff.service.organization.OrganizationServiceImpl;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RestController;
 
-@Log4j2
+import java.util.List;
+import java.util.Objects;
+
 @RestController
-public class OrganizationController implements BrokersApi {
+public class OrganizationController implements OrganizationsApi {
 
-  private BrokerServiceImpl service;
+  private final OrganizationServiceImpl organizationService;
```

Comment:
let's always depends on interfaces, not implementations
```suggestion
  private final OrganizationService organizationService;
```

Reply:
this has not been resolved